src/mmchecking_list.cpp

```diff
@@ -649,28 +649,28 @@ void TransactionListCtrl::OnMouseRightClick(wxMouseEvent& event)
         : wxPLURAL("&Permanently delete selected transaction...", "&Permanently delete selected transactions...", selected));
     if (is_nothing_selected) subGlobalOpMenuDelete->Enable(MENU_TREEPOPUP_DELETE2, false);
     subGlobalOpMenuDelete->AppendSeparator();
-    subGlobalOpMenuDelete->Append(MENU_TREEPOPUP_DELETE_VIEWED, !m_cp->isTrash_ ? _("Delete all transactions in current view...") : _("Permanently delete all transactions in current view..."));
+    subGlobalOpMenuDelete->Append(MENU_TREEPOPUP_DELETE_VIEWED, !m_cp->isTrash_ ? _("Delete &all transactions in current view...") : _("Permanently delete &all transactions in current view..."));
     if (!m_cp->isTrash_) {
-        subGlobalOpMenuDelete->Append(MENU_TREEPOPUP_DELETE_FLAGGED, _("Delete Viewed \"Follow Up\" Transactions..."));
-        subGlobalOpMenuDelete->Append(MENU_TREEPOPUP_DELETE_UNRECONCILED, _("Delete Viewed \"Unreconciled\" Transactions..."));
+        subGlobalOpMenuDelete->Append(MENU_TREEPOPUP_DELETE_FLAGGED, _("Delete Viewed “&Follow Up” Transactions..."));
```

Comment:
Miust admit, I don't like using the alternatoive double quotes here and prefer escaping the 'normal' quotes.

Reply:
Is there an operating system which breaks using these quotation marks?

Mozilla standard is to use these quotes now, which an example shown below:
https://transvision.mozfr.org/?recherche=Send+websites+a+%E2%80%9CDo+Not+Track%E2%80%9D+signal+that+you+don%E2%80%99t+want+to+be+tracked&repo=all_projects&sourcelocale=en-US&locale=en-US&search_type=strings

It is understood that Apple, Google and Microsoft uses these new quotes too.

Furthermore, they also use the 1-character ellipsis `…`.

Reply:
If you use double quotation marks you MUST leave the escape character `\` to indicate to the compiler that the inner marks do not terminate the string. It will not compile as it is currently written.

`_("Delete Viewed \"Follow Up\" Transactions...")`

Using single quotes doesn't require any escape character:

`_("Delete Viewed 'Follow Up' Transactions...")`